CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ MINOR changes (backwards-compatible):
 
 PATCH changes (bugfixes):
 
-*
+* Updated documentation and tests to remove the requirement that `/dev/shm` is executable.
+This requirement was actually removed in v3.0.0, but we didn't document it.
```

Comment:
Maybe edit the 3.0 release notes to mention that?

Reply:
done